comments.php

```diff
@@ -28,7 +28,8 @@
 		<h2 class="comments-title">
 			<?php
 				printf( // WPCS: XSS OK.
-					_nx( 'One thought on &ldquo;%2$s&rdquo;', '%1$s thoughts on &ldquo;%2$s&rdquo;', get_comments_number(), 'comments title', 'twentyseventeen' ),
+					/* translators: 1: comments count 2: Post title */
+					_n( 'One thought on &ldquo;%2$s&rdquo;', '%1$s thoughts on &ldquo;%2$s&rdquo;', get_comments_number(), 'twentyseventeen' ),
```

Comment:
This section should done like in the `_s` PR. https://github.com/Automattic/_s/pull/1012/files